packages/react-portal/src/virtualParent/elementContains.ts

```diff
@@ -0,0 +1,32 @@
+import { getParent } from './getParent';
+/**
+ * Similar functionality to `element.contains` DOM API for use with out of order DOM elements that
+ * checks the virtual parent hierarchy. If a virtual parents exists, it is chosen over the actual parent
+ *
+ * @returns true if the child can find the parent in its virtual hierarchy
+ */
+export function elementContains(parent: HTMLElement | null, child: HTMLElement | null): boolean {
+  if (!parent || !child) {
+    return false;
+  }
+
+  let isContained = false;
+  if (parent === child) {
+    isContained = true;
```

Comment:
```suggestion
    return true;
```

_nit_: can we use early return there?

Reply:
it's a coding style thing I guess, I can do it